sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/TransmissionTarget.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.appcenter.analytics;
+
+import java.util.Map;
+
+public class TransmissionTarget {
```

Comment:
I think this one should be AnalyticsTransmissionTarget since it's about tracking events which is an analytics feature.